db/migrate/20151203230940_add_date_type_to_content_view_erratum_filter_rules.rb

```diff
@@ -0,0 +1,5 @@
+class AddDateTypeToContentViewErratumFilterRules < ActiveRecord::Migration
+  def change
+    add_column :katello_content_view_erratum_filter_rules, :date_type, :string, :default => "issued", :null => false
```

Comment:
We "default" to updated everywhere else in the UI and when we show dates for errata we show the updated on date which implies to me that is the go to behavior and "issued" is the edge case.